docs/source/en/model_doc/owlvit.mdx

```diff
@@ -80,6 +80,8 @@ This model was contributed by [adirik](https://huggingface.co/adirik). The origi
 
 [[autodoc]] OwlViTFeatureExtractor
     - __call__
+    - post_process
+    - post_process_one_shot_object_detection
```

Comment:
Just a nit, I see the method is called "image guided detection", but the postprocess method is called "one shot detection", I guess they both mean the same thing.

Can we use the same term for both?

Reply:
Sure, I just wanted to keep it shorter but post_process_image_guided_detection is less confusing.